littlelines/otwarchive
# AO3-5886​: Restrict ability to delete comments or mark comments as spam

# Pull Request Checklist


## Issue
https://otwarchive.atlassian.net/browse/AO3-5886

## Purpose
Forbids admins without appropriate roles to delete and mark comments as spam.

## Testing Instructions
Sign is as admin without appropriate roles and check that the "spam" and "delete" buttons do not exist on comments.

Sign is as an admin with correct roles. Verify the "spam" and "delete" buttons exists, and these buttons function as expected.

## References
Are there any other relevant issues / PRs / mailing lists discussions related to this?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

(If you have an account on JIRA, please include the same name in the "Full name" field on your JIRA profile,
so we can find you and assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

(If you do not fill this in, we will use your GitHub account name and will use "they" to avoid making assumptions about your gender.)
